src/argus_htmx/templates/htmx/incidents/_table.html

```diff
@@ -0,0 +1,121 @@
+<table id="table">
+    {% block table %}
+        <thead>
+        <tr>
+            <th>actions</th>
+            <th>id</th>
+            <th>timestamp</th>
+            <th>status</th>
+            <th>severity</th>
+            <th>source</th>
+            <th>description</th>
+            <th>ack</th>
+            <th>tag</th>
+        </tr>
+        </thead>
+        <tbody>
+        {% for incident in page.object_list %}
+            <tr>
+                <td> X</td>
```

Comment:
`<td> X</td>` should also be a block.

Reply:
Open question if the tag itself should be in the block or not, if yes it would be easy to add classes/ids to the tag. It would also make it easier to replace the table with  div-in-div soo,, pros/cons...

Reply:
Anyway, each cell should have their own id at least.